virt/virt-4-11-release-notes.adoc

```diff
@@ -105,6 +105,15 @@ Deprecated features are included in the current release and supported. However,
 Removed features are not supported in the current release.
 
 //CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} {VirtVersion} removes support for link:https://nmstate.io/[nmstate] including the following objects:
```

Comment:
Are the objects examples of objects that are not supported, or are they the only objects not supported? If they're examples (and all nmstate objects are no longer supported), there should be a comma after `[nmstate]`.